seatunnel-connectors-v2/connector-http/connector-http-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/http/source/HttpSourceReader.java

```diff
@@ -337,14 +334,16 @@ public void internalPollNext(Collector<SeaTunnelRow> output) throws Exception {
             if (pageInfoOptional.isPresent()) {
                 noMoreElementFlag = false;
                 PageInfo info = pageInfoOptional.get();
+                if (!Strings.isNullOrEmpty(this.httpParameter.getBody())) {
+                    this.rawBody = this.httpParameter.getBody();
+                }
```

Comment:
Could move the behavior of set `rawBody` value into constructer?

Reply:
@Hisoka-X Please take a look at your earliest convenience